dart.rb

```diff
@@ -26,30 +27,6 @@ class Dart < Formula
     end
   end
 
-  devel do
```

Comment:
Would if be useful for folks to keep the `head do` as an alternative to a separate dev recipe? I guess the main difference would be that you don't need to worry about conflicts as much with `--head`.

Reply:
I'm not a dart user, but in other formulae I've found HEAD to be very useful, exactly because of the conflicts and dependencies issues caused by a separate -dev formula.

Reply:
@Stargator What do you think?

Reply:
TL/DR - As a user, I would prefer head.

This is where it gets tricky for me. The docs clearly state that head builds the "development cutting edge" ie head of master.

devel clearly fit the use better as it targeted dev releases instead of the head of master.

Dropping support for devel just made how people use head more inconsistent. 🙄

For Dart, we depend on users being able to have stable builds installed as well as the dev releases. It helps them quickly determine when something broke through test and comparison.

For that reason alone, as a user I would use head to avoid conflicts.

I have every version of Dart installed via homebrew from 1.24.3 to now. I used to also have the dev releases installed, but now I wait until after a stable release for testing.